parsely/src/main/java/com/parsely/parselyandroid/ParselyTracker.java

```diff
@@ -755,9 +755,7 @@ protected String doInBackground(Void... params) {
 
         @Override
         protected void onPostExecute(String advertId) {
-            adKey = advertId;
-            deviceInfo = collectDeviceInfo();
-            deviceInfo.put("parsely_site_uuid", adKey);
```

Comment:
Question (❓): Just to confirm that, why was that deleted? My guess is because it is just being a duplicate of [dInfo.put("parsely_site_uuid", uuid);](https://github.com/Parsely/parsely-android/blob/652ccbec41975239f3981289d94a40335f2495b2/parsely/src/main/java/com/parsely/parselyandroid/ParselyTracker.java#L647), right?

FYI: Because this change was included as part of this 98c1dd6583b71e165ebb1cf2dbeea3bf0e5a85ee refactor commit, which was mainly about removing the `adKey` field, it kind of made it harder for me to understand. Although you did add this on the description, I just want to be sure of it (🤷 + 😊): `It makes codebase easier to understand and eliminates multiple sources for 'parsely_site_uuid'.`

Reply:
Yes, this code was duplicating [dInfo.put("parsely_site_uuid", uuid)](https://github.com/Parsely/parsely-android/blob/652ccbec41975239f3981289d94a40335f2495b2/parsely/src/main/java/com/parsely/parselyandroid/ParselyTracker.java#L647) from `collectDeviceInfo`.